.github/workflows/test.yml

```diff
@@ -846,7 +846,7 @@ jobs:
 
       - name: Lint
         # Filters some workspaces out:
-        # - `cli` because it runs golangci-lint and this Job doesn't have all the dependencies for that.
+        # - `cli` because (no reason anymore)
```

Comment:
Should we stop filtering out `cli` then?

Reply:
separate PR, I don't know if linting is setup there at all

Reply:
actually `lint` script is gone from there, so we can safely remove this `!cli`. Done